### DISCUSSION TITLE
Versi mac compile apakah bisa?

### DISCUSSION THREAD

**Discussion by Stackopoy** [Arya-f4/worldshellfinder#5] [Q&A]
Kira-kira bisa ga bang bikin executable hasil compile-annya ke versi mac?

**Comment by Aryma-f4** [comment]
buat versi mac compile sendiri bang, karena aku ga punya device buat native compile-annya. Sebenarnya dulu pernah ada bang Architecture Darwin buat dirun ke mac, cuman entah kenapa ketika aku compile di windows dan waktu di run di laptop temenku kagak bisa. Makanya saya disable dulu bang gitu. 🙏
